### ISSUE TITLE
would you release the training code in the future?

### ISSUE THREAD

**Issue by jqliu09** [SUDO-AI-3D/zero123plus#83]

**Comment by eliphatfs** [comment]
Though not official, you can try the finetune code from https://github.com/TencentARC/InstantMesh 
I cannot release the training code for non-technical reasons..

**Comment by jqliu09** [comment]
> Though not official, you can try the finetune code from https://github.com/TencentARC/InstantMesh I cannot release the training code for non-technical reasons..

Thanks! I will have a detailed look along your instruction. And thanks again for providing such a good work~

**Comment by Mrguanglei** [comment]
@eliphatfs  Hello, now I am also training this instantmesh, but the test set uses the zero123plus code, which causes an error when testing. I would like to know how to make his test set based on OpenLRM's rendering script, because zero23 has a fixed view, but instantmesh has a random view input for its training data set.
